### PR TITLE
Improve docs performance

### PR DESCRIPTION
**Pull request by bashmish**
Uses new `speedy-links` which fixes Flash of inAccurate coloR Theme (FART) and improves perf overall.

Tricky to document, not sure where to put it, so for now just added a comment in the code about being able to disable.

### CODE REVIEW

File: mdx-layout-core/doc/index.mdx [hunk 1]
```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { mdx } from '@mdx-js/react';
 import { Description } from '~/props';
 import { CoreLayout } from '~/mdx-layout-core';
-export default CoreLayout;
+import { CustomCoreLayout } from '~/doc-layout/src/CustomCoreLayout';
```
Comment (gqio) [inline mdx-layout-core/doc/index.mdx:5]: What about SpeedyCoreLayout ? To keep the name in case of we change the lib underneath ?

**Review by gqio — APPROVED** [review]
LGTM

**Review by gqio — COMMENTED** [review]
Just wondering why not using by default speedy link this way everyone would beneficiate directly from ... and maybe expose `BaseCoreLayout` or `SimpleCoreLayout`

File: mdx-layout-core/doc/index.mdx (hunk 1 above)
Comment (bashmish) [reply mdx-layout-core/doc/index.mdx:5]: right now these 3 wrappers are private for "self-testing" dockit-react

export publicly `SpeedyCoreLayout` in addition to `CoreLayout` might be a good idea indeed
do you think such API sells better than asking to read docs in dockit-core about speedy-links?

I might make another PR with this if we agree

**Comment by bashmish** [comment]
> Just wondering why not using by default speedy link this way everyone would beneficiate directly from ... and maybe expose `BaseCoreLayout` or `SimpleCoreLayout`

answered in a thread, let's continue in it
https://github.com/divriots/dockit-react/pull/118#discussion_r839396518

File: mdx-layout-core/doc/index.mdx (hunk 1 above)
Comment (gqio) [reply mdx-layout-core/doc/index.mdx:5]: That was my take too

Comment (bashmish) [reply mdx-layout-core/doc/index.mdx:5]: let's say I stop arguing and agree with you
so you say all layout, from dockit-core layout to React and Vue should do it by default and release as a new major?

Comment (georges-gomes) [reply mdx-layout-core/doc/index.mdx:5]: I would say yes.
I don't know if it needs to be a major if there is no API change and it's all internal to `CoreLayout`.
Either way I'm find with this idea.

Comment (bashmish) [reply mdx-layout-core/doc/index.mdx]: I made it a default

**Pull request merged**